cfme/services/myservice/ui.py

```diff
@@ -106,15 +106,17 @@ class details(Tab):  # noqa
     @View.nested
     class provisioning(Tab):  # noqa
         results = SummaryTable(title='Results')
-        plays = SummaryTable(title='Plays')
+        plays = Table('.//table[./thead/tr/th[contains(@align, "left") and '
```

Comment:
**QUESTION** i presume this doesn't need a version pick?

Reply:
Yes, I just made this widget more common. It works on 5.8 and 5.9.